Dockerfile.mustgather

```diff
@@ -0,0 +1,9 @@
+FROM registry.ci.openshift.org/ocp/4.14:must-gather AS builder
+WORKDIR /go/src/github.com/openshift/secrets-store-csi-driver-operator
+COPY . .
```

Comment:
The builder container does not build anything. You can just `COPY hack/gather /usr/bin` in the real container below.

Reply:
It would be worth fixing LSO too.

Reply:
Ack, updated, and LSO fix is in https://github.com/openshift/local-storage-operator/pull/420